### PR TITLE
Fixing explanation related to the --disableupdate

### LINKED ISSUE #16801: Self-hosted runners --disableupdate should be passed during registering phase
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#controlling-runner-software-updates-on-self-hosted-runners

### What part(s) of the article would you like to see updated?

`run.sh` should not accept `--disableupdate`. You should specify it during the registration phase.

There is an open [PR](https://github.com/actions/runner/pull/1781/) which will warn if you pass `--disableupdate` during the `run.sh`, hinting that you should configure your runner with that flag.

### Additional information

This update is the result of the [comment](https://github.com/actions/runner/issues/1604#issuecomment-1086561724) on the issue in the runner repository.

### PR DESCRIPTION
**Pull request by nikola-jokic**
### Why:
The documentation states that you should pass `--disableupdate` during the `run.sh`, but it will throw a warning after the [runner PR](https://github.com/actions/runner/pull/1781) is merged. `--disableupdate` flag should be passed during the registration phase.

Closes #16801 

### What's being changed:
Explanation and example when you should pass the `--disableupdate`

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@nikola-jokic Thanks for opening a pull request! We've triaged this issue for technical review by a subject matter expert :eyes:

**Review by thboop — DISMISSED** [review]
Hey 👋 . As a maintainer of the `actions/runner` I can say this is correct from a technical standpoint, but would love if someone on the docs team could review the verbiage and get this merged whenever it is convenient for y'all.

**Review by ethomson — DISMISSED** [review]

File: content/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ The {% data variables.product.prodname_actions %} service will then automaticall
 
 By default, self-hosted runners will automatically perform a software update whenever a new version of the runner software is available.  If you use ephemeral runners in containers then this can lead to repeated software updates when a new runner version is released.  Turning off automatic updates allows you to update the runner version on the container image directly on your own schedule.
 
-If you want to turn off automatic software updates and install software updates yourself, you can specify the `--disableupdate` parameter when starting the runner.  For example:
+If you want to turn off automatic software updates and install software updates yourself, you can specify the `--disableupdate` flag when registering your runner using `config.sh`. For example:
```
Comment (skedwards88) [inline content/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners.md:61]:
```suggestion
To turn off automatic software updates and install software updates yourself, specify the `--disableupdate` flag when registering your runner using `config.sh`. For example:
```

File: content/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners.md [hunk 2]
```diff
@@ -58,10 +58,10 @@ The {% data variables.product.prodname_actions %} service will then automaticall
...
 
 ```shell
-./run.sh --disableupdate
+./config.sh --url https://github.com/octo-org --token example-token --disableupdate
```
Comment (skedwards88) [inline content/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners.md:64]: nit style suggestion
```suggestion
./config.sh --url <em>https://github.com/octo-org</em> --token <em>example-token</em> --disableupdate
```

**Review by skedwards88 — COMMENTED** [review]
Thank you for this addition! I made some minor style suggestions that I will go ahead and incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**